elements/openhpc/pre-install.d/91-install-ohpc-repo

```diff
@@ -6,8 +6,5 @@ fi
 set -eu
 set -o pipefail
 
-workspace=$(dirname $0)
-
 [ -d /etc/yum.repos.d ] || exit -1
-cp -n $workspace/OpenHPC.repo /etc/yum.repos.d/
-cp -n $workspace/OpenHPC-updates.repo /etc/yum.repos.d/
+yum install -y https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm
```

Comment:
Does this install the updates repo too?

Reply:
It does indeed! The content of the file looks like this:
```
[OpenHPC]
name=OpenHPC-1.3 - Base
baseurl=http://build.openhpc.community/OpenHPC:/1.3/CentOS_7
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-OpenHPC-1

[OpenHPC-updates]
name=OpenHPC-1.3 - Updates
baseurl=http://build.openhpc.community/OpenHPC:/1.3/updates/CentOS_7
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-OpenHPC-1

[OpenHPC-Sources]
name=OpenHPC-1.3 - Sources
baseurl=http://build.openhpc.community/OpenHPC:/1.3/sources/CentOS_7
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-OpenHPC-1
enabled=0
```